products/claim-appeal-status/document-status/engineering/drop-offs-after-failures-spike.md

```diff
@@ -0,0 +1,244 @@
+their document that failed to get to VBMS via Lighthouse (Type 2).
+
+#### Recommended Events
+Currently the presence of Type 2 errors do not fire off events, however it could be interesting to record an event when the user navigates to CST and sees it. This would require adding a new event (maybe named `claims-upload-silent-failure`) that includes the number of failed documents. Then implement the second funnel below.
+
+#### Recommended Funnels
+Type 1 Drop-off (navigation away from page)
+1. User Navigates to CST Files page or Evidence Request Page
+1. User uploads file(s) (fires off `claims-upload-start` event)
+1. One or more file(s) fail to upload showing Type 1 Unknown Error message (fires off `claims-upload-failure` event)
+1. User navigates away from CST Files page or Evidence Request Page
+
+Type 2 Drop-off (navigation away from CST - Status Tab or Files Tab)
+1. User Navigates to CST Status Tab
+1. User sees Type 2 Error (fires off new `claims-upload-silent-failure`)
+1. User navigates away OR navigates to `/files-we-couldnt-receive` page
+
+---
+
+## 2. Background & Context
+
+### Problem Statement
+- We want to measure what happens after a Veteran encounters a failure message in the document submission flow
+- There is ambiguity around what an "exit" means: does it indicate abandonment (a negative outcome), or is it expected behavior when Veterans are instructed to resubmit offline (particularly for Type 2 errors)?
+- Currently, exits could reflect two very different things — discouraged users giving up, or users appropriately following offline instructions
+
+### Current State
+
+- **Type 2 Errors & Offline Flow**: Type 2 errors instruct Veterans to resubmit documents offline, which means some exits after these errors are expected and appropriate user behavior.
+
+### Goals
+- Meaningfully differentiate between abandonment and expected offline resubmission behavior
+- Assess whether GA funnel analysis or another approach can reliably distinguish between these cases
+- This work ties to Epic #120479 – Document Status Refinements and Clarifications, specifically supporting the Analytics & Metrics Refinements goal
+
+---
+
+## 4. Technical Findings
+
+### 4.1 Google Analytics Funnel Analysis Capabilities
+
+#### Current Setup
+- **Existing Analytics/Tracking**:
+  - claims-upload-start
+  - claims-upload-cancel
+  - claims-upload-failed
+  - claims-upload-success
+  - page paths (status tab, files tab, needed-from-you, etc.,)
+
+#### Feasibility Assessment
+- GA funnels track can track abandonment drop-offs after errors. What it cannot do is determine why the veteran decided to move on (logout)
+- To get more granularity on the number of documents uploads success/failed (type 1 only), we can update the current recorded events to include that data
+- Type 2 errors do not currently
+
+### 4.3 Platform Analytics Analysis [TBD]
```

Comment:
i think you forgot to remove this? There's a Data Classification Challenges labeled 4.3